.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,42 @@
+name: Manual NPM Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseType:
+        description: 'Release type - major, minor or patch'
+        required: true
+        default: 'patch'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone Repository
+        uses: actions/checkout@v2
+      - name: Setup Node version
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.15.1
```

Comment:
We should update the node version to latest LTS (including in `.nvmrc`)

Reply:
Unfortunately, this action (`setup-node`) doesn't support keywords like `lts`, see https://github.com/actions/setup-node/pull/58. I can bump Node version to match current LTS, but until the PR is merged we will need to do it manually.